backend/src/main/java/hu/flowacademy/meteo/controller/MeasurmentController.java

```diff
@@ -12,14 +11,15 @@
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-public class TestController {
+public class MeasurmentController {
 
-    private final TestService testService;
+    private final MeasurmentService measurmentService;
 
-    @GetMapping("test/{id}")
-    public MeasurmentDto findLastByStationId(@PathVariable Long id) {
-        MeasurmentDto testMeasurment= MeasurmentDto.toDto(testService.findLastByStationId(id));
+    @GetMapping("latest/{id}")
```

Comment:
Az id használata így félrevezető, arra az entityre szokott utalni, amit a controller kezel, ami itt a measurement. A megadott id itt egy station, míg maga a 'latest' utal a measurement-re. Ezt az id-t inkább kérd be queryparamként.